CONTRIBUTING.md

```diff
@@ -93,6 +93,8 @@ If you've found something in the content or the website that should be updated,
 ### :hammer_and_wrench: Pull requests
 A [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) is a way to suggest changes in our repository.
 
+REST content is autogenerated and should not be modified by mortals.
```

Comment:
Imagine you're trying to figure out how to use the message to decide what you should / shouldn't be changing, does your text help?

I'd want `rest` to be a link or list of links. It isn't really useful as-is.

Certainly as an end user, I don't think this is particularly actionable text.

It might be a link to a /find/ search, or something else.

I mean, it isn't technically false, since that's apparently what I was told, but I don't think it helps much.

n.b. I filed this because I was told to, I'm just an end user like you, I don't have any ownership stake in this project, so please take my input w/ a grain of salt.

On a more stylistic level, you might want to use the word `note`, or an emoji, or something.